packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -15,9 +16,25 @@ export const SUCCESS_ICON = 'clipboard-checked';
 export const ERROR_ICON = 'clipboard-x';
 export const DEFAULT_STATUS = 'idle';
 
-export default class HdsCopyButtonComponent extends Component {
+export interface HdsCopyButtonSignature {
+  Args: {
+    isFullWidth?: boolean;
+    isIconOnly?: boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onError?: (...args: any[]) => void;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onSuccess?: (...args: any[]) => void;
+    size?: HdsCopyButtonSizes;
+    targetToCopy?: HdsCopyButtonItemToCopy;
+    text: string;
```

Comment:
the type for this should come from the `Button` signature (the property is forwarded to it)